packages/stores/src/account/types.ts

```diff
@@ -48,14 +49,18 @@ export const defaultMsgOpts: OsmosisMsgOpts = {
     gas: 280000,
     shareCoinDecimals: 18,
   },
-  swapExactAmountIn: {
+  splitRouteSwapExactAmountIn: (numPools: number) => ({
+    type: "osmosis/poolmanager/split-route-swap-exact-amount-in",
+    gas: 30_000 * numPools,
+  }),
+  swapExactAmountIn: (numPools: number) => ({
     type: "osmosis/gamm/swap-exact-amount-in",
-    gas: 250000,
-  },
-  swapExactAmountOut: {
+    gas: 25_0000 * numPools,
+  }),
+  swapExactAmountOut: (numPools: number) => ({
     type: "osmosis/gamm/swap-exact-amount-out",
-    gas: 250000,
-  },
+    gas: 25_0000 * numPools,
+  }),
```

Comment:
This too

Reply:
How does it handle gas?

Reply:
It will be the same object as this one. We'd only have a MessageComposer instead of a type to replace the amino classes. On the other hand, if the dev omits the gas, it's possible to simulate and get an estimated fee